requirements.txt

```diff
@@ -9,6 +9,7 @@ pytz==2015.7
 requests==2.9.1
 requests-toolbelt==0.6.0
 rfc3987==1.3.4
+strict-rfc3339==0.7
```

Comment:
Yep, I found out through an arduous search that the date-time validation inside `jsonschema` requires it but silently fails if it's not present.